runtime/dart_isolate.cc

```diff
@@ -280,8 +283,136 @@ std::weak_ptr<DartIsolate> DartIsolate::CreateRootIsolate(
   return (*root_isolate_data)->GetWeakIsolatePtr();
 }
 
+Dart_Isolate DartIsolate::CreatePlatformIsolate(Dart_Handle entry_point,
+                                                Dart_Port port_id,
+                                                const char* debug_name,
+                                                bool errors_fatal,
+                                                char** error) {
+  *error = nullptr;
+  PlatformConfiguration* platform_config = platform_configuration();
+  FML_DCHECK(platform_config != nullptr);
+  PlatformIsolateManager* platform_isolate_manager =
+      platform_config->client()->GetPlatformIsolateManager();
+  if (platform_isolate_manager->IsShutdown()) {
+    // Don't set the error string. We want to silently ignore this error,
+    // because the engine is shutting down.
+    FML_LOG(ERROR) << "CreatePlatformIsolate called after shutdown";
```

Comment:
Remove or reduce the priority of these logs if they can happen due to timing when running valid code in the user's app